### DISCUSSION TITLE
[Projects Beta] Reconsider the shortcut key of "add an item"

### DISCUSSION THREAD

**Discussion by creasty** [community/community#5835] [Projects and Issues]
## Why

Developers most likely assign Ctrl-Space or Cmd-Space as the hotkey of their app launcher (such as Alfred).

<img width="526" alt="Screen Shot 2021-09-28 at 1 43 26" src="https://user-images.githubusercontent.com/1695538/134951665-f64fe8e4-28ce-49a6-bab8-0056b5f9ba98.png">

**Comment by azenMatt** [comment]
Thanks for the feedback @creasty - we avoided the default Alfred shortcut (option + space) for this reason! 

We felt that ctrl + space wasn't currently utilised, but happy to be proved wrong if lots of people have mapped that to an app launcher. 

Please upvote above if you are also facing this issue! 😄

**Reply by creasty** [reply]
Oh okay, that sounds fair.

I thought ctrl-space was the default but I guess I’ve changed it long time ago…
(I use the ctrl modifier to minimize finger movements from the home position. For the same reason I use diamond cursor keys, and hardy ever use arrow keys, delete and return keys.)

**Reply by tylerneu** [reply]
For what it's worth, I have Alfred mapped to cmd-space and moved Spotlight to ctrl-space.

**Reply by kolomeetz** [reply]
Also `Ctrl`+`Space` is [a default shortcut to change the input language](https://support.apple.com/en-ie/guide/mac-help/mchlp1406/mac) in Mac OS.

**Reply by vergilcw** [reply]
This also tripped me up. I have used a few different launchers in the past, and given how many of them use Ctrl + Space as the primary hotkey, even if I try a new one that has a different hotkey I usually end up switching it to Ctrl+Space. 
Aside from Albert, here are a few other launchers where this is the default binding:
- Synapse
- Ulauncher
- Cerebro
- "Control Space" (browser tab switcher)
